src/AllocatorConduitExample.sol

```diff
@@ -82,16 +88,18 @@ contract AllocatorConduitExample is IAllocatorConduit {
     // --- functions ---
 
     function deposit(bytes32 ilk, address asset, uint256 amount) external ilkAuth(ilk) {
+        address buffer = registry.buffers(ilk);
+        // Implement the logic to deposit funds from the Buffer into the FundManager
```

Comment:
Can you add the `transferFrom` and `transfer` calls in deposit and withdraw?

Reply:
Done.